src/server_manager/web_app/data_formatting.spec.ts

```diff
@@ -0,0 +1,52 @@
+/*
+  Copyright 2020 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import * as i18n from './data_formatting';
+
+// Jasmine with Node only supports 'en'
+if (Intl.NumberFormat.supportedLocalesOf(['en', 'fr', 'fa', 'es', 'ru']).length === 5) {
+  describe('getFormattedDataAmountParts', () => {
+    it('extracts the unit string and value separately', () => {
+      const english = i18n.getFormattedDataAmountParts(0, 'en');
+      expect(english.unit).toEqual('B');
+      expect(english.value).toEqual('0');
+
+      const spanish = i18n.getFormattedDataAmountParts(2, 'es');
+      expect(spanish.unit).toEqual('B');
+      expect(spanish.value).toEqual('2');
+
+      const french = i18n.getFormattedDataAmountParts(1.5 * 10 ** 9, 'fr');
+      expect(french.unit).toEqual('Go');
+      expect(french.value).toEqual('1,5');
+
+      const farsi = i18n.getFormattedDataAmountParts(133.5 * 10 ** 6, 'fa');
+      expect(farsi.unit).toEqual('مگابایت');
+      expect(farsi.value).toEqual('۱۳۳٫۵');
+    });
+  });
+
+  describe('formatBytes', () => {
+    it('Formats data amounts', () => {
+      expect(i18n.formatBytes(10 * 10 ** 9, 'en')).toEqual('10 GB');
```

Comment:
Can you add a KB test so that we cover that code path?

Reply:
It's in https://github.com/Jigsaw-Code/outline-server/pull/780/commits/49ce60d8d96a8d1d434992c60e01f04c2a37af65#diff-858c37f4a836374e98fb54a760e5a4099b33a6ae04b41c2079b60c588eeb492cR31, I'm not sure why the view isn't showing it

Reply:
It seems the KB test was added to the `formatBytesParts` above.